post_user.sql

```diff
@@ -0,0 +1,49 @@
+-- phpMyAdmin SQL Dump
+-- version 5.0.2
+-- https://www.phpmyadmin.net/
+--
+-- Hostiteľ: 127.0.0.1
+-- Čas generovania: Pi 09.Okt 2020, 22:37
+-- Verzia serveru: 10.4.14-MariaDB
+-- Verzia PHP: 7.3.22
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
```

Comment:
use a [migration](https://symfony.com/doc/current/bundles/DoctrineMigrationsBundle/index.html) rather than manually doing this

Also you probably want to add a migration to copy the current authors to the new table